### PR TITLE
Add options in makeflow_graph_log to support specifying paths for gnuplot and convert

### PR DESCRIPTION
**Pull request by sghuang19**
```
makeflow_graph_log [options] <makeflowlog> <imagefile>
```

Use `-g` to specify path to `gnuplot`, `-c` to specify path to `convert`. `PATH` is used by default.

### CODE REVIEW

**Comment by sghuang19** [comment]
#2863

File: makeflow/src/makeflow_graph_log [hunk 1]
```diff
@@ -47,7 +61,7 @@ do
    then
 	  true
    else
-	  echo "$0: could not find $prog in your PATH"
+	  echo "$0: could not find $prog in the PATH variable or in the specified path"
```
Comment (btovar) [inline makeflow/src/makeflow_graph_log:64]: Let's change this whole if to:
```sh
if ! which $prog >& /dev/null
then
   echo ...
  exit 1
fi
```

File: makeflow/src/makeflow_graph_log [hunk 2]
```diff
@@ -10,17 +10,31 @@
...
 while [ $# -gt 0 ]
 do
 	case $1 in
 	-h |--help )
 		show_help
 		exit 1
 		;;
+	-g)
+		shift
+		gnuplot=$1
```
Comment (btovar) [inline makeflow/src/makeflow_graph_log:32]: "$1"

File: makeflow/src/makeflow_graph_log [hunk 3]
```diff
@@ -10,17 +10,31 @@
...
+		;;
+	-c)
+		shift
+		convert=$1
```
Comment (btovar) [inline makeflow/src/makeflow_graph_log:36]: "$1"

**Review by btovar — CHANGES_REQUESTED** [review]

**Review by btovar — CHANGES_REQUESTED** [review]

**Pull request merged**